packages/plain-object/docs/api/README.md

```
@@ -1,8 +1,8 @@
-**@httpx/plain-object v2.1.2**
+**@httpx/plain-object v2.1.3**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)